### DISCUSSION TITLE
Why isn't there a google oauth api bindings crate?

### DISCUSSION THREAD

**Discussion by Jamzy01** [Byron/google-apis-rs#461] [Q&A]
I would like a google oauth bindings crate for a tauri app I'm working on. I'm logging in using oauth and using that access token with your google drive v3 crate, which is all working fine. But on the login page, I want to show the user their email, name and profile picture, so they actually know which google account they are logged into. In the app's current state, it will be easier for me (and way more secure) to just retrieve this data from rust so I don't actually have to pass the access token to js. But I can't seem to find any [oauth2](https://developers.google.com/accounts/docs/OAuth2) crates in the [master crates list](https://byron.github.io/google-apis-rs/). I'm not exactly sure why it's not there since it's in the etc folder [here](https://github.com/Byron/google-apis-rs/blob/main/etc/api/oauth2/v2/oauth2-api.json), plus it's in the [apis.json](https://github.com/Byron/google-apis-rs/blob/main/apis.json).

**Comment by Byron** [comment]
The [oauth2 API is ignored](https://github.com/Byron/google-apis-rs/blob/d404113bee056a1635f71f449d5bcd3d6e09b4a4/etc/api/shared.yaml#L42) as apparently it didn't manage to generate working code.

This could have changed by now, maybe I will get to that towards the end of the year.
